village/src/components/SmurfForm.js

```diff
@@ -10,9 +11,22 @@ class SmurfForm extends Component {
     };
   }
 
+  // button
   addSmurf = event => {
     event.preventDefault();
     // add code to create the smurf using the api
+    const url = 'http://localhost:3333/smurfs';
+    const smurf = { ...this.state }
+
+    axios.post(url, smurf)
+      .then(res => {
+        this.setState({
+          smurf: res.data
+        })
+        window.location.reload();
```

Comment:
You can use setState instead of the window.location.reload to update you smurfs state.